README.md

```diff
@@ -74,6 +70,12 @@ Go to the Folder where want to copy this repo and type the following cmds in you
 - Twitter: [@HenaryKhuraijam](https://twitter.com/HenaryKhuraijam)
 - LinkedIn: [Henary Khuraijam](https://www.linkedin.com/in/henary-khuraijam)
```

Comment:
- [ ] This link is currently broken, when I try to click on it, it shows a 404 error page, please kindly provide the appropriate link as it is not professional to leave broken links in your readme file 👍

Reply:
Thank you, done.